src/Views/gagepage.html

```diff
@@ -225,13 +225,27 @@ <h4 class="modal-title">StreamStats Gage Page</h4>
-                        <label for="logScale" ng-if="vm.formattedDailyFlow.length > 0"> Log Scale</label>
-                        <highchart id="chart1" config="vm.chartConfig"></highchart>
-                    </div>
+                        <div class="well well-sm" style="background-color:#EBF0F5;" ng-if="vm.chartConfig">
+                            <div class="plot-filters">
+                                <input type="checkbox" ng-if="vm.formattedFloodFreq.length > 0 || vm.formattedAltFloodFreq.length > 0" id="toggleFloodStats" name="toggleFloodStats" ng-model="vm.showFloodStats" ng-change="vm.toggleFloodStats()" checked>
+                                <label for="toggleFloodStats" ng-if="vm.formattedFloodFreq.length > 0 || vm.formattedAltFloodFreq.length > 0" > Show Flood Statistics:</label>
+                                <select id="gageFloodStatsSelect" ng-if="vm.formattedFloodFreq.length > 0 || vm.formattedAltFloodFreq.length > 0 " style="width:10%" name="gageFloodStatsSelect" ng-model="vm.selectedFloodFreqStats" ng-options="floodFreqStat.name for floodFreqStat in vm.allFloodFreqStats" ng-change="vm.chooseFloodStats()" ng-disabled="vm.showFloodStats == false"></select>&nbsp;&nbsp;&nbsp;                      
+                                <input type="checkbox" ng-if="vm.formattedPeakDates.length > 0" id="togglePeaks" name="togglePeaks" ng-model="vm.peaksOnYear" ng-change="vm.togglePeakYear()" checked>
+                                <label for="togglePeaks" ng-if="vm.formattedPeakDates.length > 0"> Plot Peaks on Latest Year&nbsp;&nbsp;</label>
+                                <input type="checkbox" ng-if="vm.formattedDailyFlow.length > 0" id="logScale" name="logScale" ng-model="vm.logScale" ng-change="vm.toggleLogLinear()" checked>
+                                <label for="logScale" ng-if="vm.formattedDailyFlow.length > 0"> Log Scale</label>&nbsp;&nbsp;
+                                <button ng-click="vm.resetZoom()" style="height:80%">Reset Zoom</button><br>
+                                <!-- <button ng-click="vm.updateShadedStats()" style="height:80%">Update Shaded Stats</button><br> -->
+                                <!-- <label for="dischargeStart"> Date Range:</label>
+                                <input date-range-picker class="form-control date-picker" type="text" style="width:20%; height:80%" id="dischargeStart" length=10 ng-model="vm.dailyDatesOnly" min="vm.startAndEnd[0]" max="vmstartAndEnd[1]" options="vm.dailyDatesOnly"> -->
+                                <label for="dischargeStart"> Start Date:</label>
+                                <input type="text" style="width:11%; height:80%" id="dischargeStart" length=10 placeholder="MM-DD-YYYY">
+                                <label for="dischargeEnd"> End Date:</label>
+                                <input type="text" style="width:11%; height:80%" id="dischargeEnd" length=10 placeholder="MM-DD-YYYY">
```

Comment:
Thoughts on using the [date input selector](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/date)?

We should also populate these input fields with the default start/end date when the page loads.

```suggestion
                                <input type="date" style="width:11%; height:80%" id="dischargeStart">
                                <label for="dischargeEnd"> End Date:</label>
                                <input type="date" style="width:11%; height:80%" id="dischargeEnd">
```

Reply:
Defaults are populated! I'm definitely open to using the date input selector instead, but I was confused looking at examples so went for this one instead.